src/constants.js

```diff
@@ -1,6 +1,6 @@
 export const ColumnTypes = {
   CATEGORICAL: "categorical",
-  CONTINUOUS: "continuous"
+  CONTINUOUS: "numerical"
```

Comment:
This impacted the ability to use some of our datasets, since their metadata still refers to `"continuous"`.  Was it intentional?

Reply:
We wanted the string "continuous" to change to "numerical", but not to impact the logic or datasets. This is totally on me for not being clear. We can either change everything to numerical OR keep the strings and the logic/datasets separate. I vote for the latter.

Reply:
Good catch - sorry I missed this initially! #110